src/Maker/MakeTwigExtension.php

```diff
@@ -59,12 +66,29 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             AbstractExtension::class,
             TwigFilter::class,
             TwigFunction::class,
+            'App\\Twig\\Runtime\\'.$name.'Runtime'
         ]);
 
         $generator->generateClass(
             $extensionClassNameDetails->getFullName(),
             'twig/Extension.tpl.php',
-            ['use_statements' => $useStatements]
+            ['use_statements' => $useStatements, 'runtime_class_name' => $name.'Runtime::class']
```

Comment:
Related to my comment below: we could use `'runtime_class_name' => $runtimeClassNameDetails->getShortName()` here. Then, add the `::class` part in the template itself.